json-schema/src/main/java/com/regnosys/rosetta/generator/jsonschema/JsonSchemaEnumGenerator.xtend

```diff
@@ -0,0 +1,70 @@
+package com.regnosys.rosetta.generator.jsonschema
+
+import com.google.inject.Inject
+import com.regnosys.rosetta.rosetta.RosettaEnumValue
+import com.regnosys.rosetta.rosetta.RosettaEnumeration
+import java.util.ArrayList
+import java.util.List
+import java.util.Map
+
+class JsonSchemaEnumGenerator {
+
+	@Inject extension JsonSchemaGeneratorHelper
+	
+	def Map<String, ? extends CharSequence> generateEnumDefinitions(List<RosettaEnumeration> enumList) {
+		val result = newHashMap
+
+		for (RosettaEnumeration enumeration : enumList) {
+			val enumDefinitionContents = enumeration.generateEnumDefinition
+			result.put(enumeration.filename, enumDefinitionContents)
+		}
+
+		result
+	}
+
+	def String generateEnumDefinition(RosettaEnumeration enumeration) '''
+		{
+		  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Maybe better to factor this out into a variable that is reused by all json schema generators, so we can easily change dialects when we want to?